spec.bs

```diff
@@ -497,6 +506,22 @@ The specification adds the following steps to the [=HTTP fetch=] algorithm, befo
 1. If |redeem response result| is a [=network error=], return |issue response result|.
 
 
+Integration with XMLHttpRequest {#xhr}
+====================================
+
+An XMLHttpRequest has an associated {{PrivateToken}} object specifying the {{OperationType}} to execute against the request.
+
+The <dfn><code>setPrivateToken()</code></dfn> method accepts a {{PrivateToken}} |privateToken| and must do the following:
```

Comment:
Define this method with WebIDL:

```suggestion
<xmp class="idl">
partial interface XMLHttpRequest {
  undefined setPrivateToken(PrivateToken privateToken);
};
</xmp>

The <dfn><code>setPrivateToken()</code></dfn> method accepts a {{PrivateToken}} |privateToken| and must do the following:
```

Or whatever actual definition is appropriate.